paper/tex/methods.tex

```diff
@@ -14,8 +14,7 @@ \section{Methods}
 To optimally discriminate between classification techniques, there must be a performance metric, a single scalar value quantifying how appropriate a classifier is for the task at hand.
 Choosing a metric for \plasticc\ therefore is logically entwined with the challenge goals.
 The goal of \snphotcc\ was to identify good classifiers of SN Ia used for a photometric cosmology analysis, which lends itself to a metric that optimizes the purity of any resultant data set.
```

Comment:
Bizarrely, this isn't showing up correctly in the diff, but if you look at line 16 of methods.tex in both branches, the one you're proposing conflicts with @RickKessler's feedback about SNPhotCC, which was why the text in issue/65/2nddraftfeedback was changed the way it was.